tests/shell/shell_recorder/replay_tests/csv_metakey.epf

```diff
@@ -0,0 +1,30 @@
+CMD: kdb mount config.csv user:/tests/csvstorage csvstorage
```

Comment:
Thank you so much! Did you use on purpose the non-markdown variant of the shell recorder? See https://github.com/ElektraInitiative/libelektra/blob/master/tests/shell/shell_recorder/tutorial_wrapper/README.md for markdown variant.

Reply:
We were not aware of the markdown variant

Reply:
We have now converted the test cases into the Markdown variant